docs/examples/use_cases/tensorflow/yolov4/src/dali/ops.py

```diff
@@ -61,10 +61,7 @@ def select(predicate, if_true, if_false):
     joined = dali.fn.cat(if_true, if_false)
     sh = predicate * true_shape + (1 - predicate) * false_shape
 
-    st = dali.fn.cat(
-            dali.fn.slice(true_shape * (1 - predicate), start=[0], shape=[1], axes=[0]),
-            dali.fn.constant(idata=0, shape=[1])
-        )
+    st = dali.fn.stack(true_shape[0] * (1 - predicate), 0)
```

Comment:
I've tested this piece in isolation and it produces the same result as the original code while being much simpler.